netutil/interface.go

```diff
@@ -79,7 +79,10 @@ func NextBundleInterface(t *testing.T, dut *ondatra.DUTDevice) string {
 	t.Helper()
 	batch := dut.Telemetry().NewBatch()
 	dut.Telemetry().InterfaceAny().Aggregation().Batch(t, batch)
-	bundleIntfs := batch.Lookup(t).Val(t).Interface
+	bundleIntfs := make(map[string]*telemetry.Interface)
```

Comment:
I don't think you actually need to construct a map here. I think it would suffice to define the variable of a map type and let it be nil if unassigned, i.e.

var bundleIntfs map[string]*telemetry.Interface

same below

Reply:
thanks, done